packages/build/tests/core/snapshots/tests.js.md

```diff
@@ -543,32 +543,32 @@ Generated by [AVA](https://ava.li).
       debug: true␊
       dry: true␊
       featureFlags: {}␊
-      repositoryRoot: /file/path␊
+      repositoryRoot: packages/build/tests/core/fixtures/several␊
       testOpts:␊
         pluginsListUrl: test␊
         silentLingeringProcesses: true␊
     ␊
     > Current directory␊
-      /file/path␊
+      packages/build/tests/core/fixtures/several␊
     ␊
     > Config file␊
-      /file/path␊
+      packages/build/tests/core/fixtures/several/netlify.toml␊
     ␊
     > Resolved config␊
       build:␊
         command: echo test␊
         commandOrigin: config␊
-        functions: /file/path␊
+        functions: packages/build/tests/core/fixtures/several/functions␊
       plugins:␊
         - inputs: {}␊
           origin: config␊
-          package: /file/path␊
```

Comment:
Was looking through the fixture and seems like this is a local plugin. Wondering if this expected/if we should keep it this way?

Reply:
(After reviewing the plugins section and seeing a couple more of these I guess it's expected, since we're resolving from the cwd)

Reply:
Good catch! The untransformed version is:

```
  plugins:
    - inputs: {}
      origin: config
      package: ./plugin
```

`./plugin` should be resolved relative to `baseDir`, not `cwd`. Perhaps we should leave relative paths untransformed? 🤔  I'll experiment with that.

Reply:
Done in https://github.com/netlify/build/pull/2456/commits/80eebfbcbb466c39bbeee4261b66c6cca23acad7.